lib/billing/check.ts

```diff
@@ -40,11 +43,19 @@ export function check(config: ConfigInterface) {
     const client = new GraphqlClient({session});
 
     const plansArray = Array.isArray(plans) ? plans : [plans];
-    return hasActivePayment({
-      plans: plansArray,
-      client,
-      isTest,
-    });
+    if (returnObject) {
+      return getActivePayments({
+        plans: plansArray,
+        client,
+        isTest,
+      });
+    } else {
+      return hasActivePayment({
```

Comment:
It seems this is using the same query as `getActivePayments`. Would it simplify the code if we just check if `getActivePayments` returns anything instead of having a separate method?